docs/3_guides/0_submitting_proofs.md

```diff
@@ -160,12 +164,12 @@ aligned submit \
 --proof <proof_file> \
 --vm_program <vm_program_file> \
 --pub_input <pub_input_file> \
---batcher_url wss://batcher.alignedlayer.com \
+--batcher_url wss://mainnet.batcher.alignedlayer.com \
```

Comment:
Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error